src/tbp/monty/frameworks/environments/embodied_data.py

```diff
@@ -554,160 +554,59 @@ def get_good_view(
         allow_translation: bool = True,
         max_orientation_attempts: int = 1,
     ) -> bool:
-        """Policy to get a good view of the object before an episode starts.
-
-        Used by the distant agent to find the initial view of an object at the
-        beginning of an episode with respect to a given sensor (the surface agent
-        makes use of the `touch_object` method instead). Also currently used
-        by the distant agent after a "jump" has been initialized by a model-based
-        policy.
-
-        First, the agent moves towards object until it fills a minimum of percentage
-        (given by `motor_system._policy.good_view_percentage`) of the sensor's field of
-        view or the closest point of the object is less than a given distance
-        (`motor_system._policy.desired_object_distance`) from the sensor. This makes
-        sure that big and small objects all fill similar amount of space in the sensor's
-        field of view. Otherwise small objects may be too small to perform saccades or
-        the sensor ends up inside of big objects. This step is performed by default
-        but can be skipped by setting `allow_translation=False`.
-
-        Second, the agent will then be oriented towards the object so that the
-        sensor's central pixel is on-object. In the case of multi-object experiments,
-        (i.e., when `num_distractors > 0`), there is an additional orientation step
-        performed prior to the translational movement step.
+        """Invoke the GetGoodView positioning procedure.
 
         Args:
-            sensor_id: The name of the sensor used to inform movements.
-            allow_translation: Whether to allow movement toward the object via
-                the motor systems's `move_close_enough` method. If `False`, only
-                orientienting movements are performed. Default is `True`.
-            max_orientation_attempts: The maximum number of orientation attempts
-                allowed before giving up and returning `False` indicating that the
-                sensor is not on the target object.
+            sensor_id (str): The ID of the sensor to use for positioning.
```

Comment:
Why are we duplicating the type information that's available in the arguments list in the code?

Reply:
A justification would be that when reading our API docs, sometimes there's not enough context on the screen. e.g. https://api-monty.thousandbrains.org/tbp.monty.simulators.habitat.html#tbp.monty.simulators.habitat.agents.MultiSensorAgent scrolled down a bit look like 

<img width="1032" alt="Screenshot 2025-05-02 at 11 04 42" src="https://github.com/user-attachments/assets/80fa5904-ff32-4a13-bbad-4f1e8c58d719" />

Reply:
Fair enough. It would be nice if the tool would pull the typing information out, but I haven't looked at how those are generated yet.